src/integration/pages/mediaAssetPage/crossPlatformTests.js

```diff
@@ -23,23 +23,37 @@ export default () => {
     expect(timestampEl.textContent).toMatchSnapshot();
   });
 
-  const bulletedListEl = document.querySelector('main ul[role="list"]');
+  const bulletedListEl = document.querySelector('main ul[role="list"] > li');
 
   if (bulletedListEl) {
-    it('I can see the bulleted list item', () => {
+    it('I can see a bulleted list item', () => {
       expect(bulletedListEl).toBeInTheDocument();
       expect(bulletedListEl.textContent).toBeTruthy();
       expect(bulletedListEl.textContent).toMatchSnapshot();
     });
+
+    const link = bulletedListEl.querySelector('a');
+
+    if (link) {
```

Comment:
Just wondering whether this check for an anchor to be within the list item can be at the unit test level rather than here. Otherwise this will lead to these integration tests becoming bloated too.

I think one rule of thumb could be that only tests that cannot be adequately carried out at the unit test level should be added at the integration test level. .

Reply:
I'd like to open out this discussion to others as I am torn! In principle I agree with what you're saying.  However...

- We already do have unit tests for this - I included it here as a replacement for functionality which we had in Cypress tests (but removed because Cypress was not the right place for it).
- If we're expecting these integration tests to be a higher level than unit tests (and written and maintained by QA engineers too), is it better to include assertions like this?
- I don't want these tests to become bloated either, but currently it's taking 7-8 seconds to run all of the integration tests, so I don't want to prematurely optimise just yet - do we monitor performance, and review if it becomes a problem?

Should we have a chat with the wider group @andrewscfc @jroebu14 to see if we can agree on a rule of thumb for tests like this?